book/src/programmability/witness-pattern.md

```diff
@@ -1,277 +1,117 @@
 # Pattern: Witness
 
-Almost every combination of [abilities](../move-basics/abilities-introduction.md) forms a pattern in
-Move. And one of the most important ones is the _Witness pattern_ - a type that with the
-[`drop`](./../move-basics/drop-ability.md) ability. It is used to instantiate generic types with a
-concrete type parameter by providing a _proof of ownership_ of this type.
+Witness is a pattern of proving an existence by constructing a proof. In the context of programming,
+witness is a way to prove a certain property of a system by providing a value that can only be
+constructed if the property holds.
```

Comment:
I wonder if you need the first sentence -- I think it might be fine to drop it. It overlaps with the next sentence, and I'm having a hard time getting it to work just right because "an existence" makes me wonder "of what", and it's kind of the other way around, "the existence of the witness is the proof of some property, by construction", but this is essentially what you are saying in the next line.

```suggestion
In the context of programming, a witness is a way to prove a certain property of a system by providing a
value that can only be constructed if the property holds.
```